odk1-src/form-question-types.rst

```diff
@@ -1867,6 +1867,28 @@ When the :tc:`picker` appearance is added, the range widget is displayed with a
 
   range, range_integer_widget_picker, Range picker integer widget, picker, range integer widget with picker appearance, start=1;end=10;step=1
 
+.. _range-rating-widget:
+
+Range widget with rating
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+type
+  :tc:`range`
+appearance
+  :tc:`rating`
+
+When the :tc:`rating` appearance is added, the range widget is displayed with stars having equal spacing. Number of stars is calculated using `end` parameter. When the user taps on an empty star, the stars up to and including that star will be filled. If the stars don't fit in the device width, they will be wrap onto additional lines.
```

Comment:
1. is calculated using the `end` parameter (add the)
2. they will wrap onto additional lines (remove be)